api/wallet/spec/paths/withdrawals.yaml

```
@@ -44,75 +45,74 @@ get:
       required: false
     - name: createdAtFrom
       in: query
-      description: Дата создания с
+      description: Creation date from
```

Если речь о диапазоне, то вроде так более понятно
```suggestion
      description: Creation date range start
```